tests/unit/test_bake.py

```diff
@@ -246,6 +247,11 @@ def test_gpcp_bake(
             else:
                 zarr_store_full_paths = [config["TargetStorage"]["root_path"]]
 
+            # dictobj runs do not generate any datasets b/c they are not recipes
+            # so we've asserted what we can already, just move on
+            if recipes_version_ref.endswith("dictobj"):
+                return
```

Comment:
reviewer note: this couldn't have been actually producing output for `0.10.x` so I think this is fine